main.go

```diff
@@ -50,6 +54,41 @@ func init() { //nolint:gochecknoinits
 	//+kubebuilder:scaffold:scheme
 }
 
+func setupClients(cfg *rest.Config) (client.Client, client.Client, error) {
+	cli, err := client.New(cfg, client.Options{Scheme: scheme})
```

Comment:
Above `setupClients()`, or in `main()` (where seems appropriate), can you add a comment explaining why we need setup clients, reconciler, and healthcheck?